### ISSUE TITLE
Integrate with Zen-fs org

### ISSUE THREAD

**Issue by konsumer** [konsumer/web-zen-dev#11]
This repo should be in `@zenfs`.

I propose:
- @james-pre should have admin on this repo.
- I transfer ownership to @zen-fs GH org
- Add me a as a maintainer, or at least ensure I have commit & GH release rights
-  He creates `@zenfs/devices` on npm.
- I can add "publish to npm on release" CI actions (so we both have access to publishing on npm, but I have no other access) and it's fairly automatic (just make a new release)

**Comment by james-pre** [comment]
Sounds good. All the current ZenFS repos use the same approach for releases. Thanks again for your help!

**Comment by konsumer** [comment]
I think it's not really ready for release on npm, but we could do transfer now.

I think I need "create repo" permissions on zen-fs. Is there a better way to transfer?

<img width="734" alt="Screenshot 2024-11-05 at 11 56 11 AM" src="https://github.com/user-attachments/assets/0d5a3d74-d4b1-4ebd-b5b8-4ab25b67db3b">

**Comment by james-pre** [comment]
Yeah, I agree.

**Comment by james-pre** [comment]
@konsumer It may be easier for me to set up a new repo with all of the tooling and such, then I can just add you and you can commit the code. Does that work for you?

**Comment by konsumer** [comment]
yep, you could fork it yourself, then add me to it, then I can archive original. That way we keep all the commit-history. We'll lose issues, but I can add them back.

**Comment by konsumer** [comment]
I do similar on [node-raylib](https://github.com/RobLoach/node-raylib). I have maintainer-rights, but no access to secrets, and 3 of us (twuky, RobLoach, and I) all can publish if we need to, and we share the issue queue.

**Comment by konsumer** [comment]
Actually @james-pre I think there is a better way that will keep issues. I add you as owner, then you transfer, and make sure I can access after.

**Comment by james-pre** [comment]
Unfortunately, I don't have permissions to view the settings. I'll go ahead and make a new repo since that seems to be the easiest.

**Comment by james-pre** [comment]
@konsumer Please reach out to me on Discord (or, if you really need to, email).

**Comment by konsumer** [comment]
@james-pre I do not know how to reach you by either of these.

**Comment by james-pre** [comment]
For [Discord](https://discord.com), you can send a friend request to me using my username, `alsovo`.
